client/src/js/samples/components/__tests__/General.test.js

```diff
@@ -65,11 +59,21 @@ describe("mapStateToProps()", () => {
             isolate: "Isolate Foo",
             locale: "Bar",
             paired: false,
-            srna: false,
             gc: "31.2 %",
             count: "13.2 m",
             lengthRange: "50 - 100",
-            subtractionId: "baz"
+            subtractionId: "baz",
+            libraryType: "Normal"
         });
     });
+
+    it.each([["normal", "Normal"], ["srna", "sRNA"], ["amplicon", "Amplicon"]])(
+        "state.library_type(%s) should equal props.libraryType(%s)",
+        (a, b) => {
+            const wrapper = shallow(<SampleDetailGeneral />);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['wrapper' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4674330)